packages/angular/src/app/pages/planning-task-details/task-form/task-form.component.ts

```diff
@@ -29,18 +29,22 @@ import { DxTextAreaModule } from 'devextreme-angular/ui/text-area';
 export class TaskFormComponent implements OnChanges {
   @Input() task: Task;
 
+  @Input() contentByScreen: { xs: number, sm: number }
```

Comment:
Why do we use the input to set xs and sm values?

Reply:
Because pass here https://github.com/DevExpress/devextreme-ui-template-gallery/pull/97/files#diff-05bf450067ccf330250a8002372399ba51776d45da4623406af38c5187a27638R40 and https://github.com/DevExpress/devextreme-ui-template-gallery/pull/97/files#diff-94066e4a65d058aaea661efa7926ee71ade36a71bcdff49490d12e8e38925245R135